### DISCUSSION TITLE
Generate m-to-m relations with sea-orm-cli

### DISCUSSION THREAD

**Discussion by dorianim** [SeaQL/sea-orm#1066] [Q&A] [answered]
Hey,

I am currently trying to create a many-to-many relation with sea-orm. I have defined according migrations with three tables, user, group and group_member.
I then ran the migrations and used `sea-orm-cli generate entity` to generate the entities. However, this only generates relations for user -> group_member -> group but not for user -> group as described [here](https://www.sea-ql.org/SeaORM/docs/relation/many-to-many/). Am I doing something wrong, or is this just not supported?

Regards,
Dorian

**Comment by billy1624** [comment, marked as answer]
Hey @dorianim, I double checked the CLI and found there is some issue blocking it to generate `impl Related` for m-to-n relation. I'll now file a issue for it and draft a PR to solve it. Thanks for the report!!

**Reply by billy1624** [reply]
Related issues and PR
- https://github.com/SeaQL/sea-orm/issues/1074
- https://github.com/SeaQL/sea-orm/pull/1075

**Reply by dorianim** [reply]
Tanks!

**Comment by swwu** [comment]
In the meantime, are there any workarounds to using many-to-many relations while relying on auto-generated files created via `sea-orm generate entity`, other than modifying those files directly?
